build_helpers/publishPackage.sh

```diff
@@ -1,35 +1,70 @@
 #!/bin/bash
+# example usages:
+# ./publishPackage.sh --branch=v1.0.x
+# ./publishPackage.sh --branch=v1.1.x --latest
```

Comment:
To run the script through npm, user will have to do:
`npm run publish-package -- --branch v1.1.x --latest`

The ` -- ` immediately after the script name is used for separating the params passed to the npm command itself with the params passed to `publishPackage.sh`